python/lsst/daf/base/exampleLib.cc

```diff
@@ -0,0 +1,44 @@
+#include <pybind11/pybind11.h>
+#include <pybind11/operators.h>
+
+#include "lsst/daf/base/DateTime.h"
+
+using namespace lsst::daf::base;
+
+namespace py = pybind11;
+
+PYBIND11_PLUGIN(_exampleLib) {
+    py::module mod("_exampleLib", "example baseLib module with pybind11");
+
+    py::class_<DateTime> cls(mod, "DateTime");
+
+    py::enum_<DateTime::Timescale>(cls, "Timescale")
+        .value("TAI", DateTime::Timescale::TAI)
+        .value("UTC", DateTime::Timescale::UTC)
+        .value("TT", DateTime::Timescale::TT)
+        .export_values();
+
+    py::enum_<DateTime::DateSystem>(cls, "DateSystem")
+        .value("JD", DateTime::DateSystem::JD)
+        .value("MJD", DateTime::DateSystem::MJD)
+        .value("EPOCH", DateTime::DateSystem::EPOCH)
+        .export_values();
+
+    cls.def(py::init<long long, DateTime::Timescale>(), py::arg("nsecs") = 0LL, py::arg("scale") = DateTime::TAI)
+        .def(py::init<long long, DateTime::Timescale>())
```

Comment:
Is it practical to support keyword arguments for constructors? How about overloaded constructors?